db/migrate/20190923181352_create_external_registry_bikes.rb

```diff
@@ -0,0 +1,26 @@
+class CreateExternalRegistryBikes < ActiveRecord::Migration
+  def change
+    create_table :external_registry_bikes do |t|
+      t.string :type, null: false, index: true
+      t.belongs_to :country, null: false, index: true
+
+      t.string :serial_number, null: false, index: true
```

Comment:
Do we actually need to index `serial_number`? I don't think we actually search on that column normally, do we?

Reply:
We do when searching for exact matches:

```rb
# app/models/external_registry_bike.rb L15-17 (5933eeb3)

def find_or_search_registry_for(serial_number:)
  matches = ExternalRegistryBike.where(serial_number: serial_number)
  return matches if matches.any?
```
<sup>[[source](https://github.com/bikeindex/bike_index/blob/5933eeb3/app/models/external_registry_bike.rb#L15-L17)]</sup>

Reply:
We shouldn’t actually be doing this. Every search should be for the normalized serial. That’s one of our promises.

Reply:
Sorry I didn’t catch this before!

Reply:
Good catch thanks -- updated in 7b012932